test/rpc/endpoints/get_network_test.exs

```diff
@@ -0,0 +1,44 @@
+defmodule Soroban.RPC.CannedGetNetworkClientImpl do
+  @moduledoc false
+
+  @behaviour Soroban.RPC.Client.Spec
+
+  @impl true
+  def request(_endpoint, _url, _headers, _body, _opts) do
+    send(self(), {:request, "RESPONSE"})
+
+    {:ok,
+     %{
+       friendbot_url: "https://friendbot-futurenet.stellar.org/",
+       passphrase: "Test SDF Future Network ; October 2022",
+       protocol_version: "20"
+     }}
+  end
+end
+
+defmodule Soroban.RPC.Endpoints.GetNetworkTest do
```

Comment:
I think the modulename should be `Soroban.RPC.GetNetworkTest` since the `Endpoints` context does not really exist. Apply this change for previous modules too